python/ray/data/_internal/execution/streaming_executor.py

```diff
@@ -449,6 +445,7 @@ def _debug_dump_topology(topology: Topology, log_to_stdout: bool = True) -> None
     for i, (op, state) in enumerate(topology.items()):
         logger.get_logger(log_to_stdout).info(
             f"{i}: {state.summary_str()}, "
-            f"Blocks Outputted: {state.num_completed_tasks}/{op.num_outputs_total()}"
+            f"Blocks Outputted: {state.num_completed_tasks}/{op.num_outputs_total()}\n"
+            f"{op.metrics.as_dict()}"
```

Comment:
nit, maybe print the metrics in a different loop. mixing them together may make the logs harder to read.

Reply:
Also, we moved the interval out of this part of the code, do we still want to log this in an interval?